src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -12117,7 +11841,7 @@ public static void Main()
     )
     // Fields
     .field public static initonly class Program/'<>c' '<>9'
-    .field public static class '<>f__AnonymousDelegate0`2'<int32, int32> '<>9__1_0'
+    .field public static class '<>f__AnonymousDelegate0' '<>9__1_0'
     // Methods
     .method private hidebysig specialname rtspecialname static 
```

Comment:
>// (4,12): error CS0656: Missing compiler required member 'System.Runtime.CompilerServices.DecimalConstantAttribute..ctor' [](http://example.com/codeflow?start=16&length=122)

The error looks unexpected, we don't need to synthesize the attribute because one is explicitly applied

Reply:
But we actually synthesize it for the delegate type. `_baseParameterForAttributes` is not used for delegates because we don't want to propagate most attributes to them. I have removed this test, because it should error on `DecimalConstantAttribute` not being defined, but I guess `MakeTypeMissing` is not enough. I could test this using empty compilation, but than it just errors on the type missing, so it's not very interesting.